python/ray/tune/trial.py

```diff
@@ -531,6 +531,19 @@ def init_logdir(self):
             self.logdir = create_logdir(self._generate_dirname(), self.local_dir)
         else:
             os.makedirs(self.logdir, exist_ok=True)
+
+        # Add restored checkpoint
```

Comment:
This is probably the only change in behavior. Here we would add old existing checkpoints back into the checkpoint manager and possibly delete them as well.

Reply:
ah I see.
This is during resume right?
Is the current behavior do nothing?

Reply:
Yes, the current behavior is to just leave existing checkpoints as is.

Generally we should probably add _all_ checkpoints in the logdir to the checkpoint manager and not just the latest - but that's out of scope for this PR